### PR TITLE
feat(ParticipantSelectable): add component for participant search results and bulk selection

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Ref #12800
* Extract `searched` behaviour` from `<Participant />` component into a new one
* Migrate component to NcListItem (template) and TS (script)
* Rewrite color scheme for the component

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image
image | image

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

**Comment by Antreesy** [comment]
#12819 covers both cases now, so will update this PR later

**Comment by ShGKme** [comment]
Actual?

**Comment by Antreesy** [comment]
> Actual?

No, need to rewrite now (to be checkbox, not an anchor

File: src/components/BreakoutRoomsEditor/SelectableParticipant.vue [hunk 1]
```diff
@@ -60,28 +67,83 @@ export default {
...
+			default: true,
+		},
 	},
 
-	emits: ['update:checked'],
+	emits: ['update:checked', 'click-participant'],
+
+	setup() {
+		// Toggles the bulk selection state of this component
+		const isSelectable = inject('bulkParticipantsSelection', false)
```
Comment (DorraJaouad) [inline src/components/BreakoutRoomsEditor/SelectableParticipant.vue:81]: isBulkSelection
```suggestion
		const isBulkSelection = inject('bulkParticipantsSelection', false)
```

**Review by DorraJaouad — APPROVED** [review]
Tested, it works 🦅

**Pull request merged**